### DISCUSSION TITLE
OneUI 6 cant go below 24 hz?

### DISCUSSION THREAD

**Discussion by knowledgeispower101** [tribalfs/GalaxyMaxHzPub#144] [Q&A] [answered]
My Fold4 worked beautifully on OneUI 5.1.1 on the v8.1.27, allowing me to do 10 - 120 hz. However since updating to one ui 6.0, every time I set 10 to 120 the app resets it to 24 to 120 after I leave any menu. Is this a bug?

**Comment by tribalfs** [comment, marked as answer]
You need to allow restricted settings.  Please check  here how.
https://github.com/tribalfs/GalaxyMaxHzPub/releases

**Reply by knowledgeispower101** [reply]
Thank you!!
